AudioPlayer/AudioPlayer/event/PlayerEventProducer.swift

```diff
@@ -70,7 +70,7 @@ class PlayerEventProducer: NSObject, EventProducer {
         case progressed(CMTime)
         case endedPlaying(Error?)
         case interruptionBegan
-        case interruptionEnded
+        case interruptionEnded(Bool)
```

Comment:
I'd be great if you add a label for this param, like `.interruptionEnded(shouldResume: Bool)`, so it's clear that the Bool is not about the fact that the interruption ended, but what to do after.